fontes/avaliador-sintatico/avaliador-sintatico.ts

```diff
@@ -683,6 +683,7 @@ export class AvaliadorSintatico implements AvaliadorSintaticoInterface<SimboloIn
         let incrementar = null;
         if (!this.verificarTipoSimboloAtual(tiposDeSimbolos.PARENTESE_DIREITO)) {
             incrementar = this.expressao();
+            this.verificarSeSimboloAtualEIgualA(tiposDeSimbolos.INCREMENTAR, tiposDeSimbolos.DECREMENTAR)
```

Comment:
A ideia é dar erro aqui? Porque isso apenas avança o símbolo se for incrementar ou decrementar.

Reply:
@leonelsanchesdasilva Não. Aqui só avança o `this.atual` caso for um desses simbolos, pois se não tiver, o símbolo atual vai ser um desses símbolos, ae quebra no consumir logo abaixo, que espera PARENTESES_DIREITO.